App.js

```diff
@@ -1,21 +1,22 @@
 import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import { createStackNavigator, StackActions, NavigationActions } from 'react-navigation';
+
+import HomeScreen from './screens/HomeScreen';
+import InputScreen from './screens/InputScreen';
+import ResultsScreen from './screens/ResultsScreen';
+import InputManual from './components/InputManual';
 
-export default class App extends React.Component {
-  render() {
-    return (
-      <View style={styles.container}>
-        <Text>Open up App.js to start working on your app!</Text>
-      </View>
-    );
-  }
-}
 
-const styles = StyleSheet.create({
```

Comment:
That's also valid for our code, but where do all the StyleSheet go? 
I see you deleted it here, but do we leave it on the JS files or in the screens? I'm not too sure.

Reply:
We should all look into putting style sheets in a separate file for consistency. Let's look into how to do this.

Reply:
A single stylesheet will be created to simpify and clean the screens also to improve reusability and decoupling.